cli/js/compiler.ts

```diff
@@ -1047,61 +1049,85 @@ interface SourceFileMapEntry {
   typeHeaders: ReferenceDescriptor[];
 }
 
-interface CompilerRequestCompile {
+interface CompileRequest {
   type: CompilerRequestType.Compile;
   allowJs: boolean;
   target: CompilerHostTarget;
   rootNames: string[];
   configPath?: string;
   config?: string;
   unstable: boolean;
-  bundle: boolean;
   cwd: string;
   // key value is fully resolved URL
   sourceFileMap: Record<string, SourceFileMapEntry>;
 }
 
-interface CompilerRequestRuntimeCompile {
+interface BundleRequest {
+  type: CompilerRequestType.Bundle;
+  target: CompilerHostTarget;
+  rootNames: string[];
+  configPath?: string;
+  config?: string;
+  unstable: boolean;
+  cwd: string;
+  // key value is fully resolved URL
+  sourceFileMap: Record<string, SourceFileMapEntry>;
+}
+
+interface RuntimeCompileRequest {
   type: CompilerRequestType.RuntimeCompile;
   target: CompilerHostTarget;
   rootNames: string[];
   sourceFileMap: Record<string, SourceFileMapEntry>;
   unstable?: boolean;
-  bundle?: boolean;
   options?: string;
 }
 
-interface CompilerRequestRuntimeTranspile {
+interface RuntimeBundleRequest {
```

Comment:
I don't understand the difference between BundleRequest and RuntimeBundleRequest...

Reply:
`BundleRequest` - used when `deno bundle` is invoked.

`RuntimeBundleRequest` - used when `Deno.bundle()` API is called.

Reply:
I see. Maybe ApiBundleRequest might be a better name? In any case, can you put these descriptions into a code comment? (Same for CompileRequest and RuntimeCompileRequest.)

Reply:
Added comments and sticked with current names - IMO they're more descriptive than `ApiCompileRequest`